layouts/community/ortho_4x12/wanleg/config.h

```diff
@@ -12,15 +21,15 @@
 #endif
 
 // set top left key as bootloader mode escape key on 4x4 48key layout
-#if defined(KEYBOARD_4x4) && !defined(PRO_MICRO)
+#if defined(KEYBOARD_40percentclub_4x4) && !defined(PRO_MICRO)
```

Comment:
Actually, you could just use: 
```suggestion
#if defined(BOOTLOADER_QMK_DFU)
```

If you check the `bootloader.mk` file, that's what it defines for the QMK DFU.

Reply:
makes sense. i'll change it in my next PR